src/util/XMLUtils.h

```diff
@@ -21,7 +21,7 @@
  *
  */
 
-#include <platform/util/StdString.h>
+#include <p8-platform/util/StdString.h>
```

Comment:
it's not. be as specific as possible with these, so you don't get a collision if something else is in your path with the same name